src/extensions/lens-renderer-extension.ts

```diff
@@ -42,6 +42,7 @@ export class LensRendererExtension extends LensExtension {
   kubeObjectMenuItems: KubeObjectMenuRegistration[] = [];
   commands: CommandRegistration[] = [];
   welcomeMenus: WelcomeMenuRegistration[] = [];
+  workloadsOverviewDetailItems: WorkloadsOverviewDetailRegistration[] = [];
```

Comment:
Maybe we should prefix with kube.. for example `kubeWorkloadOverviewItems`?

Reply:
prefixed